user-guide/Advanced_Functionality/Swarming/Swarming.md

```diff
@@ -4,25 +4,80 @@ uid: Swarming
-> - Swarming will not be available in DataMiner Systems with a [storage per DMA setup](xref:Configuring_storage_per_DMA) (Cassandra or MySQL).
-> - Swarming will not be available in DataMiner Systems with [Failover](xref:About_DMA_Failover).
-> - Swarming will not be available in DataMiner Systems where the [*LegacyReportsAndDashboards* soft-launch option](xref:Overview_of_Soft_Launch_Options#legacyreportsanddashboards) is enabled.
+> The above capabilities are possible with limited downtime and as long as there is spare capacity.
+
+## Limitations
+
+Some functionality is currently not yet supported for the Swarming feature. These are the most important limitations at the moment:
+
+- Swarming is not available in DataMiner Systems with a [storage per DMA setup](xref:Configuring_storage_per_DMA) (Cassandra or MySQL).
+
+- Swarming is not available in DataMiner Systems with [Failover](xref:About_DMA_Failover).
+
+- Swarming is not available in DataMiner Systems where the [*LegacyReportsAndDashboards* soft-launch option](xref:Overview_of_Soft_Launch_Options#legacyreportsanddashboards) is enabled.
+
+- Swarming is currently limited to [basic elements](xref:SwarmingElements). For [bookings](xref:SwarmingBookings), swarming is supported if the *BookingSwarming* [soft-launch option](xref:SoftLaunchOptions) is enabled.
+
+> [!NOTE]
+> [Prerequisite checks](xref:EnableSwarming#running-a-prerequisites-check) are in place to prevent the enabling of the Swarming feature when non-supported objects are present. Where possible, you will also be prevented from configuring or creating these on a Swarming-enabled system.
+
+Below you can find a complete overview of the differences between a system using DataMiner 10.5.1 or higher with or without Swarming.
+
+|                                                  | 10.5.1+&nbsp; systems without Swarming | 10.5.1+ systems with Swarming  |
+|--------------------------------------------------|----------------------------------------|--------------------------------|
+| Element configuration                            | On disk (element.xml)                  | In database                    |
+| Alarm IDs                                        | Per DataMiner Agent                    | Per element                    |
+| Database per DMA                                 | Supported                              | Not supported                  |
+| Scripts & QActions using legacy alarm references | Supported                              | Not supported                  |
+| Legacy Reports & Dashboards                      | Supported                              | Not supported                  |
+| Failover                                         | Supported                              | Not supported                  |
+| Legacy Correlation                               | Not supported                          | Not supported                  |
```

Comment:
should we still mention this as this is removed entirely in 10.5.1?

Reply:
removed this one